### ISSUE TITLE
jx-convert-jenkinsfile fail when the agent is defined as a YAML block

### ISSUE THREAD

**Issue by vbehar** [jenkins-x/jx#6520]
### Summary 

[jx-convert-jenkinsfile](https://github.com/jenkins-x/jx-convert-jenkinsfile) fails on a `Jenkinsfile` with an agent defined as a YAML block.

### Steps to reproduce the behavior

Run version 0.0.1 on the following `Jenkinsfile`:

```
pipeline {
    agent {
        kubernetes {
            yaml """

"""
        }
    }
}
```

### Expected behavior

no error

### Actual behavior

```
Error parsing Jenkinsfile:  Jenkinsfile Jenkinsfile cannot be parsed. It may contain code outside of the pipeline {} block, or it may not have a pipeline {} block at all.: 4:21: literal not terminated
```

**Comment by vbehar** [comment]
FYI I found the same issue on a multi-line shell script:

```
pipeline {
    stages {
        stage('stage') {
            steps {
                container(name: 'kaniko', shell: '/busybox/sh') {
                    sh '''#!/busybox/sh
                    /kaniko/executor -c `pwd` --destination=${DOCKER_REPOSITORY}:${PREVIEW_VERSION}
                    '''
                }
            }
        }
    }
}

```

**Comment by abayer** [comment]
Pesky string quoting! Will work on this and #6521 now.

**Comment by abayer** [comment]
Actually, `container(name: 'kaniko', shell: '/busybox/sh') {` will cause its own problems, so lemme open an issue for that.

**Comment by abayer** [comment]
Fixed in v0.0.2, I hope. =)
